src/main/java/com/bridgelabz/onlinelibrarymanagementsystem/entity/Book.java

```diff
@@ -13,4 +13,10 @@ public class Book {
     int id;
     String title, author, genre, isbn;
     Date publishedDate;
+
+
+    public boolean getId() {
```

Comment:
Extremely wrong way to create a getter and setter

Reply:
resolved sir